target/linux/rockchip/image/Makefile

```
@@ -29,7 +29,7 @@ define Build/pine64-img
 	# combining boot partition, root partition as well as the u-boot bootloader
 
 	# Generate a new partition table in $@ with 32 MiB of 
-	# alignment padding for the idbloader and u-boot to fit:
+	# alignment padding for the u-boot-rockchip.bin to fit:
```

```suggestion
	# alignment padding for the u-boot-rockchip.bin (idbloader + u-boot) to fit:
```
make it bit more clear